.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Template for requesting new features
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+## Context
+
+What is the problem that you are trying to solve?
+Why is this problem relevant?
+
+## Solution
+
+(This section and the ones below are optional.)
+What are the possible solutions?
+If there are multiple alternatives, what are the benefits and drawbacks of each one?
+
+## Subtasks
+
+- [ ] If there is a solution, what are the subtasks for completing this issue?
```

Comment:
if someone proposes a feature request does this mean he/she is a proper person to decompose the task? shouldn't the decomposition be done by the one who is going to implement a solution?